submissions/kriLEBED/html_css_popup/css/style.css

```diff
@@ -0,0 +1,151 @@
+body {
+    font-family: 'Ubuntu', sans-serif; 
+    color: #858282;
+}
+li {
+    list-style-type: none;
+}
+.wrapper {
+    max-width: 1110px;
+    padding: 20px 15px;
+    margin: 0 auto;
+}
+/*menu*/
+.nav_menu {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    font-size: 24px;
+}
+.menu_items {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    border: solid 1px transparent;
+    border-radius: 10px;
+    margin: 10px;
+    padding: 5px;
+}
+.menu_items:hover {
+    border: solid 1px blue;
+    border-radius: 10px;
+}
+.menu-link {
+    text-decoration: none;
+    color: #858282;
+}
+.profile-img {
+    height: 50px;
+    width: 50px;
+    border-radius: 10px;
+}
+.menu_button {
+    background: url(../img/menu-icon.png) center no-repeat;
+    cursor: pointer;
+    overflow: hidden;
+}
+.submenu_btn {
+   opacity: 0;
+   cursor: pointer;
+   display: block;
+   width: 43px;
+   height: 43px;
+}
+/*submenu*/
+.submenu_btn:checked ~ .submenu {
+    display: block;
+}
+.submenu {
+    position: absolute;
+    top: 125px;
+    right: 150px;
+    display: none;
+    width: 33%;
+    min-width: 335px;
+    max-height: 100%;
+    min-height: 520px;
+    border: 1px solid rgba(0, 0, 0, 0.2);
+    box-shadow: 0 1px 2px 0 rgba(60, 64, 67, .30), 0 2px 6px 2px rgba(60, 64, 67, .15);
+    overflow-x: hidden;
+    background: #fff;
+    box-sizing: content-box;
+    font-size: 18px;
+}
+.submenu-items {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 20px;
+}
+.submenu-item {
+    width: calc(100% / 3);
+    margin-top: 20px;
+}
+.submenu-link {
+    text-decoration: none;
+    color: #858282;
+}
+.submenu-link:hover {
+    background-color: #DCDCDC;
+    border-radius: 20px;
+}
+.submenu-link:focus {
+    background-color: #FFE4C4;
+    border-radius: 20px;
+}
+.submenu-item a {
```

Comment:
Avoid styling by #id or HTML tag. [Read more](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, separate responsibilities:

- HTML tag for markup;
- CSS class for styling;
- Id for JS logic